crates/bevy_macro_utils/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 extern crate proc_macro;
 
 mod attrs;
+pub mod fq_std;
```

Comment:
Let's try to leave the crate better than we started. Progressing it towards being able to use `warn(missing_docs)` would be nice. If you can, please add a doc comment to the module.

Reply:
I'll give the module a once-over and see what I can do!